### PR TITLE
Specify a default queue in several SomVal steps.

### PR DESCRIPTION
**Pull request by tmooney**
I am surprised so many SomaticValidation build steps didn't already have a queue set; I'd thought this was required awhile ago.  At any rate, now `WorkflowBuilder` will specify a queue for any steps that don't have one set.

### CODE REVIEW

File: lib/perl/Genome/WorkflowBuilder/Command.pm [hunk 1]
```diff
@@ -247,6 +247,11 @@ sub operation_type_attributes {
             $attributes{$expected_attributes{$name}} = $value;
         }
     }
+
+    unless (defined $attributes{'lsfQueue'}) {
+        $attributes{'lsfQueue'} = Genome::Config::get('lsf_queue_build_worker');
+    }
```
Comment (wschroeder) [inline lib/perl/Genome/WorkflowBuilder/Command.pm]: Should this be "lsf_queue"?

Comment (tmooney) [inline lib/perl/Genome/WorkflowBuilder/Command.pm]: I believe this key needs to match the value on [line 215](https://github.com/genome/genome/pull/1487/files/155a71a73af43a4d3124e87e54d0a84249609fad#diff-bdb6fd10fa2c2b74b604776c9d33f79dR215).

Comment (tmooney) [inline lib/perl/Genome/WorkflowBuilder/Command.pm]: At any rate, it looks like changing this default will force me to update the expected output of almost twenty tests (which highlights plenty of cases not setting defaults!), so I think I'll pull this change into its own PR.

**Comment by sleongmgi** [comment]
+1

**Comment by wschroeder** [comment]
+1 🐺

**Pull request merged**